### DISCUSSION TITLE
Disable error handler model in livewire model?

### DISCUSSION THREAD

**Discussion by jeanali** [livewire/livewire#4142] [1. Help]
I have a function to create a post everything is found with livewire code and it is working just fine, but when I open inspect element and change the function name from HTML I got an error handler that is the function does not exist!

```
<button type="button" wire:click="create()" class="btn btn-success">Add</button>
```

I know it is normal to show that the function does not exist! but I need to disable this model and show some alert or something neater!

**Comment by joshhanley** [comment]
@jeanali please don't open two discussions with exactly the same thing. I have deleted your second discussion. Please keep the discussion to this one.

Thanks!

**Comment by joshhanley** [comment]
@jeanali I'm not exactly sure what you are asking for, can you explain it a bit further? Why wouldn't you just make sure that the method exists?
